types/forge-viewer/index.d.ts

```diff
@@ -1115,6 +1116,50 @@ declare namespace Autodesk {
             function getHtmlTemplate(url: string, callback: (error: string, content: string) => void): void;
             function lerp(x: number, y: number, t: number): number;
 
+            class FragmentList {
```

Comment:
Is `FragmentList` a value exported by the namespace `Private`, or should this be declared as an `interface`? The project link at the top of this file points to a page that no longer exists and the only reference I can find is this: https://forge.autodesk.com/en/docs/viewer/v7/reference/Private/ which does not include `FragmentList`.

Reply:
Good point. Will fix this.